pearcej/cpp4python
# Add Turtles folder into Figures folder (cpp4python)

# Description
As in the issue described the image in chapter 8 was not showing that was because there was no folder called `Turtles` along image called `cc_polygon.png` Now I copied the `Turtles` folder from the restructured version and put it to `Figures` folder.


## Related Issue



fix #153 
## How Has This Been Tested?
1. Make changes locally.
2. Verify the changes.
![image](https://github.com/pearcej/cpp4python/assets/117699634/8b7566a3-1932-49db-87b2-cf4b8990da79)
